sources/packages/api/src/services/institution/institution.service.ts

```diff
@@ -768,11 +775,13 @@ export class InstitutionService extends RecordDataModelService<Institution> {
   /**
    * Update institution.
    * @param institutionId
+   * @param auditUserId user who is making the changes.
```

Comment:
not part of the PR, it would be good if you could add brief details to other comments too

Reply:
Thanks for adding the comments